source/catalog.html

```diff
@@ -0,0 +1,259 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Каталог: Кэт энерджи</title>
+</head>
+
+<body class="page__body">
+  <header class="page-header">
+    <div class="page-header__logo-container">
+      <button class="page-header__logo-toggle">
+        <span class="visually-hidden">Открыть меню</span>
+      </button>
+    </div>
+    <nav class="page-header__nav site-nav">
+      <ul class="site-nav__list">
+        <li class="site-nav__item">
+          <a class="site-nav__link site-nav__link--active" href="">главная</a>
+        </li>
+        <li class="site-nav__item">
+          <a class="site-nav__link" href="">Каталог продукции</a>
+        </li>
+        <li class="site-nav__item">
+          <a class="site-nav__link" href="">Подбор программы</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="page-main">
+    <div class="page-main__title-block">
+      <h1 class="page-main__title">Каталог продукции</h1>
+    </div>
+    <section class="page-main__catalog catalog">
+      <h2 class="visually-hidden">Содержимое каталога продукции</h2>
+      <ul class="catalog__list card">
+        <li class="catalog__item">
+          <h3 class="card__title">
+            <a class="card__link-title header-title" href="#">Cat Energy PRO 500 г</a>
+          </h3>
+          <div class="card__information">
+            <span class="card__discription">Масса</span>
```

Comment:
```suggestion
            <span class="card__description">Масса</span>
```
опечатка здесь и ниже)